app/models/item.js

```diff
@@ -166,6 +170,26 @@ export default cloudinaryUrl.extend({
     }
   }),
 
+  hasSingleLocation: Ember.computed('packagesLocations.[]', function(){
+    if(this.get('packagesLocations')){
+      return this.get('packagesLocations').length === 1;
+    }
```

Comment:
I feel this is better 👉  ` Ember.isEqual(this.get('packagesLocations').get('length'), 1)`.
Let me know if you have any input on this.